packages/local_auth/lib/local_auth.dart

```diff
@@ -90,6 +90,19 @@ class LocalAuthentication {
         'authenticateWithBiometrics', args);
   }
 
+  /// Returns true if auth was cancelled successfully.
```

Comment:
Mention in the comments that this only works for Android.

Reply:
Resolved.

Reply:
Resolved.